### PR TITLE
Add Turkish Ad Hosts

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/113

### CODE REVIEW

File: filters/regional/filter_40_TUR_TurkishAdHosts/configuration.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "TUR: Turkish Ad Hosts",
+    "description": "Necessary hosts file to block tracking (and possibly advertising) networks in commonly used mobile applications (and possibly WEB pages) in Turkey.",
+    "homepage": "https://github.com/symbuzzer/Turkish-Ad-Hosts",
+    "sources": [
+      {
+        "name": "Turkish Ad Hosts",
+        "source": "https://github.com/symbuzzer/Turkish-Ad-Hosts/blob/main/adguard/filter.txt",
```
Comment (Alex-302) [inline filters/regional/filter_40_TUR_TurkishAdHosts/configuration.json:8]: Please use raw version, instead on GitHub page
https://raw.githubusercontent.com/symbuzzer/Turkish-Ad-Hosts/main/adguard/filter.txt

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

File: locales/en/filters.json [hunk 2]
```diff
@@ -150,5 +150,9 @@
 	{
 		"hostlist.39.name": "The Blocklist Project - Ads List",
 		"hostlist.39.description": "The collection of web addresses that have been identified as hosting or serving unwanted advertisements."
+	},
+	{
+		"hostlist.40.name": "TUR: Turkish Ad Hosts",
+		"hostlist.40.description": "Necessary hosts file to block tracking (and possibly advertising) networks in commonly used mobile applications (and possibly WEB pages) in Turkey."
```
Comment (slavaleleka) [inline locales/en/filters.json:156]: do not forget to upload the base strings to crowdin

Comment (zloyden) [reply locales/en/filters.json:156]: @Sergey-Lyapin so what about this?

**Comment by zloyden** [comment]
@aalpatkina please review these strings https://github.com/AdguardTeam/HostlistsRegistry/pull/168/files#diff-530696e9394703290c9625954c438c8d3f498efbdd9148f948809d2850913c73R155-R156.

**Comment by Sergey-Lyapin** [comment]
Localizations will be added separately by @ameshkov

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**